packages/perseus-core/src/utils/split-perseus-item.ts

```diff
@@ -3,18 +3,18 @@ import _ from "underscore";
 import {getPublicWidgetOptionsFunction} from "../widgets/core-widget-registry";
 import {getUpgradedWidgetOptions} from "../widgets/upgrade";
 
-import type {PerseusItem} from "../data-schema";
+import type {PerseusItem, PerseusRenderer} from "../data-schema";
 
 /**
- * Return a copy of a Perseus item with rubric data removed (ie answers)
+ * Return a copy of a PerseusRenderer with rubric data removed (ie answers)
  *
- * @param originalItem - the original, full Perseus item (which includes the rubric - aka answer data)
+ * @param original - the original, full PerseusRenderer (which includes the rubric - aka answer data)
  */
-export default function splitPerseusItem(
-    originalItem: PerseusItem,
-): PerseusItem {
-    const item = _.clone(originalItem);
-    const originalWidgets = item.question.widgets ?? {};
+export function splitPerseusRenderer(
```

Comment:
Could `splitPerseusRenderer` be in its own file? It's slightly strange for it to be exported from a file called `split-perseus-item.ts`. Moving it to its own file would also limit the blast radius of the dependency cycle with group-util.ts